Services/JobRunner/ObjectModel/ScheduledJobTable.cs

```diff
@@ -0,0 +1,52 @@
+﻿// *********************************************************************
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License
+// *********************************************************************
+
+using DataX.Utilities;
+using System;
+using System.Composition;
+using System.Threading.Tasks;
+using DataX.Contract;
+
+namespace JobRunner

```

Comment:
Should this be DataX.Jobrunner?

Reply:
Would like to keep it JobRunner since we would want this to be generic at some point in the near future and perhaps create a NuGet such that other teams can use it. Thoughts?

Reply:
ok